### PR TITLE
feat: add student pulse

### PR DESCRIPTION
**Pull request by priyadarshini-silpa**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes

### Description

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [ ] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: client/src/components/widgets/Pulse.js [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
 
 import makeStyles from '@mui/styles/makeStyles';
-import { Card, CardContent, Grid, Typography, Button } from '@mui/material';
+import {
+  Card,
+  CardContent,
+  Grid,
+  Typography,
+  Button,
+  ButtonBase,
```
Comment (120EE0692) [inline client/src/components/widgets/Pulse.js:10]: @priyadarshini-silpa remove unused components.

File: client/src/components/widgets/Pulse.js [hunk 2]
```diff
@@ -23,29 +34,38 @@ const Pulse = () => {
             {console.log()}
```
Comment (120EE0692) [inline client/src/components/widgets/Pulse.js:34]: remove console.log()

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
@priyadarshini-silpa 
-> center the poll image in mobile view.
-> add a circle behind the image.

**Review by 120EE0692 — APPROVED** [review]

**Review by rutajdash — COMMENTED** [review]
@priyadarshini-silpa @120EE0692 are the sizing and formatting issues that were occurring with calendar side by side been all fixed?

**Comment by 120EE0692** [comment]
> @priyadarshini-silpa @120EE0692 are the sizing and formatting issues that were occurring with calendar side by side been all fixed?

@rutajdash yes.

**Pull request merged**